workflow-core/src/main/java/com/squareup/workflow1/RenderContext.kt

```diff
@@ -173,10 +172,13 @@ fun <PropsT, StateT, OutputT, ChildRenderingT>
  *
  * @param key An optional string key that is used to distinguish between identical [Worker]s.
  */
-fun <PropsT, StateT, OutputT> BaseRenderContext<PropsT, StateT, OutputT>.runningWorker(
-  worker: Worker<Nothing>,
+/* ktlint-disable parameter-list-wrapping */
+inline fun <reified W : Worker<Nothing>, PropsT, StateT, OutputT>
+    BaseRenderContext<PropsT, StateT, OutputT>.runningWorker(
+  worker: W,
```

Comment:
Without this, matching a `Worker<Nothing>` will fail because the type captured here will be `Worker<java.lang.Void>`, which is not a subtype of `Worker<Nothing>`.